app/lib/webfinger.rb

```diff
@@ -20,17 +21,28 @@ def subject
     end
 
     def link(rel, attribute)
-      links.dig(rel, attribute)
+      links.dig(rel, 0, attribute)
+    end
```

Comment:
I like how you reworked the method to still work as it did before. But did you check if it is actually used anymore?

I only had a cursory glance, but I think it might not be needed anymore.

Reply:
It's used in just one place - https://github.com/mastodon/mastodon/blob/a8330be93ee825948a17c810af33e28bfa2c56c8/app/models/remote_follow.rb#L65

Reply:
Ah, good catch. Thanks!